lib/pure/stats.nim

```diff
@@ -261,6 +292,123 @@ proc kurtosisS*[T](x: openArray[T]): float =
   rs.push(x)
   result = rs.kurtosisS()
 
+import random
```

Comment:
```suggestion
import std/random
```

Reply:
Inside `std` (and lib/pure is std too) imports shouldn't use `std/`

Reply:
Done